src/dotnet/commands/dotnet-install/InstallCommand.cs

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using Microsoft.DotNet.Cli.CommandLine;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.DotNet.Configurer;
+using Microsoft.DotNet.ShellShimMaker;
+using Microsoft.DotNet.ToolPackageObtainer;
+using Microsoft.Extensions.EnvironmentAbstractions;
+using NuGet.Build.Tasks;
+using NuGet.Configuration;
+
+namespace Microsoft.DotNet.Cli
+{
+    public class InstallCommand
```

Comment:
This class should follow the Add command pattern. We will most likely have sub-command to it beyond install tool.

Reply:
A bunch of the options here should be parsed on the sub-command.

Reply:
Will do